packages/eas-cli/src/commands/workflow/run.ts

```diff
@@ -79,8 +79,15 @@ const EXIT_CODES = {
 
 export default class WorkflowRun extends EasCommand {
   static override description = 'run an EAS workflow';
+  static override aliases = ['run'];
```

Comment:
We also have `build:run`, so I am a little concerned about customer confusion with adding the alias.